package.json

```diff
@@ -498,17 +498,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```

Comment:
As I recall they had these in there for a reason, you should probably add them back.

Reply:
Oh interesting, I'm curious to know what the reasoning behind this is. @JoeRobich why were these needed?

Reply:
> why were these needed?

I have no idea if they are needed. Our update package dependencies launch task will restore them prior to the next release. So no reason to do it here.